roles/proxmox_provision_vm/README.md

```diff
@@ -1,9 +1,98 @@
 # proxmox_provision_vm
 
-Should apply `proxmox_server` to KVM to install promoxer
+Should apply` proxmox_server` to KVM to install promoxer
 
-- `add_local_host`: adds host name and ip pair to local /etc/hosts
-- `pve_kvm_sshkeys`: defines SSH keys to be installed on KVM host
-- `kvm_start`: Start server and wait for it to start
+-` proxmox_server`: Dict of proxmox servers and credentials
+    -` key`: Hostname of proxmox server
+        -` api_host`: address of server
+        -` api_user`: username to login as
+        -` api_password`: password to login as
 
-TODO: document other vars and dicts in `tasks/main.yml` and defaults.
+-` add_local_host`: adds host name and ip pair to local /etc/hosts
```

Comment:
Please fix the markdown for all the entries in this file.

This:
```markdown
-` var`: blah blah
```
should become
```markdown
- `var`: blah blah
```